frontend/cypress/integration/signInForm.spec.js

```diff
@@ -44,43 +44,18 @@ describe('Log in form submission', () => {
   before(() => {
     // create user and log him in.
     cy.createUser(username, email, password)
-    cy.visit('login')
-    cy.get('a').contains('Login').click({ force: true })
+    cy.clearCookies()
+    cy.visit('/login')
+    cy.get('[data-cy=login-grid] a').contains('Login').click({ force: true })
   })
 
   it('should display error message on submitting form with wrong username', () => {
-    cy.stubSignInReq(false, {
-      error: 'Not Found',
-      message: 'Wrong username or password.',
-    })
     cy.signIn('nonRegUser', password)
     cy.get('.negative').should('include.text', 'Wrong username or password')
   })
 
   it('should display error message on submitting form with wrong password', () => {
-    cy.stubSignInReq(false, {
-      error: 'Not Found',
-      message: 'Wrong username or password.',
-    })
     cy.signIn(username, 'wrong_password_1234')
     cy.get('.negative').should('include.text', 'Wrong username or password')
   })
-
-  it('should display error message on submitting form for inactive account', () => {
-    cy.stubSignInReq(false, {
-      error: 'ActivationRequired',
-      message: 'Activate your account.',
-    })
-    cy.signIn('inactiveUser', password)
-    cy.get('.negative').should('include.text', 'Activate your account.')
-  })
-
-  it('should display error message on submitting form for prohibited user', () => {
-    cy.stubSignInReq(false, {
-      error: 'Prohibited',
-      message: 'Prohibited login.',
-    })
-    cy.signIn('prohibitedUser', password)
-    cy.get('.negative').should('include.text', 'Prohibited login.')
-  })
```

Comment:
Removing these because all the `stub` things had weird errors on my test runs that I had no idea how to resolve.

Reply:
You shouldn't be rolling this back! The other reason to remove this by the way is that we shouldn't be baking the http API into the e2e tests.

Reply:
Also, while it may not be showing up on the CI these stubs were definitely causing problems running e2e locally on my machine.